core-web/libs/template-builder/src/lib/components/template-builder/components/template-builder-box/template-builder-box.component.ts

```diff
@@ -72,16 +77,23 @@ export class TemplateBuilderBoxComponent implements OnChanges {
         return this.el.nativeElement;
     }
 
+    get dropdownLabel(): string {
+        return this.boxVariant === this.templateBuilderSizes.large || this.dialogVisible
+            ? this._dropdownLabel
+            : '';
+    }
+
     ngOnChanges(): void {
         this.boxVariant = getBoxVariantByWidth(this.width);
-        this.dropdownLabel =
-            this.boxVariant === TemplateBuilderBoxSize.large
-                ? this.dotMessage.get('dot.template.builder.add.container')
-                : null;
+        this._dropdownLabel = this.dotMessage.get('dot.template.builder.add.container');
     }
 
     onContainerSelect({ value }: { value: DotContainer }) {
         this.addContainer.emit(value);
         this.formControl.setValue(null);
     }
+
+    boxSizeMatches(size: TemplateBuilderBoxSize): boolean {
+        return this.boxVariant === this.templateBuilderSizes[size];
+    }
```

Comment:
Are we going to use this?

Reply:
I was testing something, I forgot to remove it.